app/src/styles.sass

```diff
@@ -42,8 +43,18 @@ input
     &[disabled], &[disabled]:hover
       border-color: tame($maincolor)
       background-color: tame($userbuttonbgcolor)
-      color: tame($useboxfgcolor)
+      color: tame($useboxfgcolor) 
 
+@media screen and (max-width: $breakpoint-tablet)
+  .dialog-box .formactions
+    padding: 0
+    flex-direction: column
+    align-items: stretch
+    margin-left: 0
+    button
+      @include userbox
```

Comment:
You don't need to repeat that here. buttons in element with formactions class have already included userbox. Moreover, it results in errors, like the font size which is smaller on narrow screens. Just remove this line.